### PR TITLE
Sn(feat): redesign

### PR DESCRIPTION
**Pull request by Suritam-Nandy**
added navbar to all the pages, but the animation isn't working for the time being.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,591 +1,390 @@
...
-                  <svg class="clip-svg">
-                    <defs>
-                      <clipPath id="clip-diamond-demo" clipPathUnits="objectBoundingBox">
-                        <polygon points="0.5 0, 1 0.5, 0.5 1, 0 0.5" />
-                      </clipPath>
-                    </defs>
-                  </svg>
+      <div class="jumbotron row">
+        <div class="col-lg-6">
+          <img src="./res/img/cars/CR16.JPG" width="100%" height="100%" />
```
Comment (Ritvick-me) [inline index.html]: Add width and height using css in a css file. Don't hardcode them with HTML.

File: index.html [hunk 2]
```diff
@@ -1,591 +1,390 @@
...
 
-                  <svg class="clip-svg">
-                    <defs>
-                      <clipPath id="clip-diamond-demo" clipPathUnits="objectBoundingBox">
-                        <polygon points="0.5 0, 1 0.5, 0.5 1, 0 0.5" />
-                      </clipPath>
-                    </defs>
-                  </svg>
-                </div>
+          <img src="res/img/logos/logo-without-bg.png " style="width: 172px;" />
```
Comment (Ritvick-me) [inline index.html]: Don't use in-line css.

File: res/css/cars.css [hunk 3]
```diff
@@ -13,7 +13,7 @@ body {
   box-sizing: border-box;
   scroll-behavior: smooth;
 }
-
+/*
```
Comment (Ritvick-me) [inline res/css/cars.css]: remove unused css

File: res/css/cars.css [hunk 4]
```diff
@@ -219,10 +219,10 @@ footer.camber-footer small span {
 
 footer.camber-footer small a {
   text-decoration: none;
-}
+} */
 
 @media (max-width: 576px) {
-  h2.camber-title {
+  /* h2.camber-title {
```
Comment (Ritvick-me) [inline res/css/cars.css]: Remove unused css from everywhere

**Review by Ritvick-me — COMMENTED** [review]
Please resolve the commented issues

**Comment by sriram-kailasam** [comment]
@Suritam-Nandy we usually don't push any commented code as it only clutters the repository. You should delete any commented or unused code before pushing. Please take care of this :)

**Pull request merged**